### DISCUSSION TITLE
Trouble with initial use! Looking for hot tips

### DISCUSSION THREAD

**Discussion by resistorsoftware** [nvim-neorg/neorg#1416] [Q&A]
I am using nvim v0.9.5 via Lazy. **checkhealth** reports all is well with **neorg**

I have not done anything to use neorg and so when I start nvim in my home directory, I am free to create a file. If I first check Neorg workspace it tells me my current workspace is the directory I am in. Fine. But I was expecting to some kind of structure by default. I have no index.norg, nor any choices on a default workspace or other workspace. 

So what is the best way to use neorg? Do I create a blank index.norg file in my home directory and just go from there? I used the command Neorg journal, picked the option today, and sure enough, it created a directory journal with the month and day as further subdirectories and my file of today's notes. So it works. 

TL:DR; I am not understanding the way to start using **neorg!**

So do I really just start off with nothing and keep on manually saving **dot norg** files and all is well? Do I have to provide some root **index.norg** file to ensure **neorg** works?

Sorry for the n00b questions, but also, the install is confusing! I went with the barest of bones installs like this:
```
{
    "nvim-neorg/neorg",
    dependencies = { "luarocks.nvim" },
    lazy = false, -- Disable lazy loading as some `lazy.nvim` distributions set `lazy = true` by default
    version = "*", -- Pin Neorg to the latest stable release
    config = true,
}
```

thinking maybe this is the culprit, to then added a more detailed configuration as per some other notes like this:

```
config = function()
      require("neorg").setup {
        load = {
          ["core.defaults"] = {},
          ["core.concealer"] = {},
          ["core.dirman"] = {
            config = {
              workspaces = {
                notes = "~/notes",
              },
              default_workspace = "notes",
            },
          },
        },
      }

      vim.wo.foldlevel = 99
      vim.wo.conceallevel = 2
    end,
```

And so now I am really confused as to why the first version of that is provided, yet the wiki offers the second version. Does it matter? 

Anyway, I am hopefully someone can steer me straight on using the neorg correctly so I am not going to regret my spelunking with it!!

**Comment by albertalef** [comment]
Im having exactly the same difficulty with the neorg. I just dont know how to use it xD

I just need some ideia of a good workflow
